auth-web/src/components/auth/account-settings/payment/AccountPaymentMethods.vue

```diff
@@ -340,6 +345,26 @@ export default defineComponent({
           state.paymentMethodChanged = false
           initialize()
           orgStore.setCurrentOrganizationPaymentType(state.selectedPaymentMethod)
+          if (state.selectedPaymentMethod === PaymentTypes.EFT) {
+            const recipientList = []
+            await orgStore.syncActiveOrgMembers()
+            state.activeOrgMembers.forEach((member) => {
```

Comment:
should we be doing this for other payment methods as well? 

Also what happens if there are too many coordinators or admins?